proto/spire/api/server/agent/v1/agent.proto

```diff
@@ -190,3 +197,13 @@ message AgentX509SVIDParams {
     // ignored. The agent X509-SVID attributes are determined by the server.
     bytes csr = 1;
 }
+
+message PushStatusRequest {
+    // Required. Serial number of the bundle used to sign Agent identity.
+    string authority_serial = 1;
```

Comment:
The idea of this serial is to keep on server database the current serial number used for registered agent.
In POC I didn't implement this, so we'll require some changes to access this data (maybe on agent endpoint) 
We can avoid this on initial implementation too...

Reply:
agree, maybe we can create an issue on spire to track all the work about persisting and recovering this information with some APIs